src/products/eventListing/template.tsx

```diff
@@ -54,7 +60,7 @@ export function EventListing({
           ></nav>
         )}
 
-        <div className="events vads-u-display--flex vads-u-flex-direction--column vads-u-padding-x--1p5 medium-screen:vads-u-padding-x--0 vads-u-padding-bottom--2">
+        <div className="events vads-u-display--flex vads-u-flex-direction--column vads-u-padding-x--1p5 tablet:vads-u-padding-x--0 vads-u-padding-bottom--2">
```

Comment:
@moanu404 Can we change this back to `medium-screen` This would line things up between 786 and 640? With this I couldn't see place where it didn't work but let me know if I'm missing something